core/templates/dev/head/pages/exploration_player/exploration_player.html

```diff
@@ -18,18 +18,59 @@
       GLOBALS.INTERACTION_SPECS = JSON.parse(
         '{{INTERACTION_SPECS|js_string}}');
     </script>
+
   {% endif %}
 
   {{dependencies_html}}
+  <style>
+    @media(max-width: 768px) {
+      /* This prevents the navbar from collapsing in the exploration page,
+      so that the exploration title can be shown. */
+      .oppia-exploration-header {
+        display: block;
+        float: right;
+        max-width: 240px;
+        overflow: hidden;
+        text-overflow: ellipsis;
+        white-space: nowrap;
+      }
+      .oppia-navbar-collapse.collapse {
+        display: block;
+      }
+    }
+    @media(max-width: 600px) {
+      .oppia-logo {
+        height: 48px;
+        margin-top: 8px;
+        margin-left: 0;
+        max-width: none;
+      }
+      .oppia-exploration-header {
+        max-width: 180px;
+      }
+      .oppia-navbar-brand-name {
+        max-width: 35px;
+        overflow: hidden;
+      }
+      .oppia-navbar-breadcrumb-icon {
+        display: none;
+      }
+    }
+    @media(max-width: 400px) {
+      .oppia-exploration-header {
+        max-width: 130px;
+      }
+    }
+  </style>
 {% endblock header_js %}
 
 {% block navbar_breadcrumb %}
   <ul class="nav navbar-nav oppia-navbar-breadcrumb" ng-controller="LearnerViewBreadcrumb">
     <li>
       <span class="oppia-navbar-breadcrumb-separator"></span>
-      <span class="protractor-test-exploration-header" itemprop="description">{{exploration_title}}</span>
+      <span class="protractor-test-exploration-header oppia-exploration-header" itemprop="description">{{exploration_title}}</span>
     </li>
-    <li ng-click="showInformationCard()" tooltip="<['I18N_PLAYER_INFO_TOOLTIP' | translate]>" tooltip-placement="bottom" style="cursor: pointer;" class="protractor-test-exploration-info-icon">
+    <li ng-click="showInformationCard()" tooltip="<['I18N_PLAYER_INFO_TOOLTIP' | translate]>" tooltip-placement="bottom" style="cursor: pointer;" class="protractor-test-exploration-info-icon oppia-exploration-info-icon">
```

Comment:
@seanlip done. i think i added it as i was intially trying to fix #2873 issues. Will take care of it.